osgp/protocol-adapter-dlms/osgp-jasper-interface/src/main/java/org/opensmartgridplatform/adapter/protocol/jasper/config/JasperWirelessConfig.java

```diff
@@ -63,6 +63,9 @@
   @Value("${jwcc.api_version}")
   private String apiVersion;
 
+  @Value("${jwcc.validity_period:0}")
+  private String validityPeriod;
```

Comment:
```suggestion
  private short validityPeriod;
```

Reply:
I tried to keep the way of setting the properties the same as it was done on other places in this class